virt/vm_networking/virt-connecting-vm-to-secondary-udn.adoc

```diff
@@ -0,0 +1,27 @@
+:_mod-docs-content-type: ASSEMBLY                               
+[id="virt-connecting-vm-to-secondary-udn"]                                    
+= Connecting a virtual machine to a secondary localnet user-defined network                                                
+include::_attributes/common-attributes.adoc[]                   
+:context: virt-connecting-vm-to-secondary-udn                                    
+                                                                
+toc::[]
+
+You can connect a virtual machine (VM) to an OVN-Kubernetes localnet secondary network by using the CLI. Cluster administrators can use the `ClusterUserDefinedNetwork` (CUDN) custom resource definition (CRD) to create a shared OVN network across multiple namespaces.
+
+[NOTE]
+====
+An OVN-Kubernetes secondary network is compatible with the xref:../../networking/multiple_networks/secondary_networks/configuring-multi-network-policy.adoc#compatibility-with-multi-network-policy_configuring-additional-network[multi-network policy API] which provides the `MultiNetworkPolicy` custom resource definition (CRD) to control traffic flow to and from VMs. You must use the `ipBlock` attribute to define network policy ingress and egress rules for specific CIDR blocks. Using pod or namespace selector policy peers is not supported.
```

Comment:
I would take this link out of here and just have it in the Additional resources.